WDL/runtime/cache.py

```diff
@@ -44,15 +62,34 @@ def get(
         opens shared flocks on all files referenced therein, which will remain for the life of the
         CallCache object.
         """
-        raise NotImplementedError()
+        from .. import values_from_json
+
+        file_path = os.path.join(self.outputs_cache_dir, f"{key}.json")
+
+        try:
+            with open(file_path, "rb") as file_reader:
+                contents = file_reader.read()
+        except FileNotFoundError:
+            return None
+        contents = json.loads(contents)
+        return values_from_json(contents, output_types)
```

Comment:
Later on we'll need to verify that all File values referenced in the cached outputs still exist before considering them valid to use; otherwise whatever wants to consume them is going to fail.

Ideally we'd also remember their modification timestamps (the newest amongst them) too to check they haven't been modified since being cached either, but that's probably not critical for the first version of this.